tests/test_alias.py

```diff
@@ -0,0 +1,582 @@
+import logging
+import random
+import string
+import time
+import requests
+
+from setup import NODE_URL, WALLET, BLOCK_WAIT_TIME, TEST_ENV
+
+
+def create_wallet():
+    # def test_create_wallet():
+    response = requests.get(NODE_URL+"/generate-wallet-address")
+    assert response.status_code == 200
+    wallet = response.json()
+    assert wallet['address']
+    assert wallet['public']
+    assert wallet['private']
+
+    response = requests.post(NODE_URL+'/add-wallet', json={
+        "custodian_address": WALLET['address'],
+        "ownertype": "1",
+        "jurisdiction": "910",
+        "kyc_docs": [
+            {
+                "type": 1,
+                "hash": "686f72957d4da564e405923d5ce8311b6567cedca434d252888cb566a5b4c401"
+            }
+        ],
+        "specific_data": {},
+        "public_key": wallet['public']
+    })
+
+    print(response.text)
+    assert response.status_code == 200
+    unsigned_transaction = response.json()
+    assert unsigned_transaction['transaction']
+    assert len(unsigned_transaction['signatures']) == 0
+    unsigned_transaction['transaction']['fee'] = 1000000
+
+    response = requests.post(NODE_URL+'/sign-transaction', json={
+        "wallet_data": WALLET,
+        "transaction_data": unsigned_transaction
+    })
+
+    assert response.status_code == 200
+    signed_transaction = response.json()
+    assert signed_transaction['transaction']
+    assert signed_transaction['signatures']
+    assert len(signed_transaction['signatures']) == 1
+
+    response = requests.post(
+        NODE_URL+'/validate-transaction', json=signed_transaction)
+    assert response.status_code == 200
+
+    if TEST_ENV == 'local':
+        response = requests.post(
+            NODE_URL + '/run-updater?add_to_chain_before_consensus=true')
+    else:
+        print('Waiting to mine block')
+        time.sleep(BLOCK_WAIT_TIME)
+
+    response = requests.get(NODE_URL+'/get-wallet',
+                            params={'wallet_address': wallet['address']})
+    assert response.status_code == 200
+    print("Wallet created with address "+wallet['address'])
+    print('Test passed.')
+    return wallet
+
+def create_token(wallet, owner , token_name,token_code, amount):
```

Comment:
do we need create_token etc for alias test?

Reply:
Yes, it isn not needed. Removed in the latest commit. Thanks